backend/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mhart/alpine-node:10.7.0 as test
+WORKDIR /usr/src/app
+COPY package.json .
+COPY yarn.lock .
+RUN yarn install
+COPY . .
```

Comment:
I don't think you'll need `COPY package.json` or `COPY yarn.lock` because it's copied here.

Reply:
As far as I know, if we removed those two lines and put the `yarn install` after the `COPY . .` then every time we changed the source code and rebuilt the container it would re-run yarn install. If we leave this as is, it will only re-run `yarn install` when we re-build the container if the `package.json` or `yarn.lock` is edited.